example/android/gradle.properties

```diff
@@ -42,3 +42,7 @@ newArchEnabled=false
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
 hermesEnabled=true
+
+# Use this property to enable or disable push notifications and firebase init
+# If set to false, firebase will not be initialized
+enableFirebase=false
```

Comment:
Why not call this `enablePushNotifications` or something similar.

Reply:
good call. i went back and forth with that trying to anticipate pulling in more firebase dependent features. updating to your recommendation!